README.md

```diff
@@ -1 +1,39 @@
-# silverstripe-tx-translator
+# silverstripe tx translator
+
+Module to assist with transifex integration. Works with Silverstripe 4.12+ and 5.0+.
+
+## Operations
+
+- Pull latest translations from transifex and merge them into yml/json/js translation files
+- Run i18nTextCollectorTask on a local website
+- Push updated source strings to transifex (optional)
+- Create github pull-requests with file changes
+
+## Requirements
+
+Transifex cli client (tx) - [https://developers.transifex.com/docs/cli](https://developers.transifex.com/docs/cli) configured API key from transifex.
+
+You must use the go version at least version 1.6+ and not the old python version.
+
+## Usage
+
```

Comment:
```suggestion

This must be run in the context of a working Silverstripe project (with database, etc). You can install it as a dev dependency within that project if you want, or you can `git clone` it into some directory somewhere - so long as when you run it the working directory in your terminal is at the project root for your Silverstripe project.

```

Reply:
Not accurate - see comment below in the autoloader

Reply:
I disagree but apparently I can't convince you so I'll just resolve the conversation.